app/uk/gov/hmrc/openidconnect/userinfo/domain/GovernmentGatewayDetails.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.openidconnect.userinfo.domain
+
+import uk.gov.hmrc.play.http.Token
+
+case class GovernmentGatewayDetails(
+                        user_id: Option[String],
+                        gateway_token: Option[Token],
```

Comment:
Ugly but those fields are not used in the code - only in tests but I think this is acceptable. They just represent the JSON so keeping them like this lets me to use built-in JSON formatter from play with one line of code. If I change the fields names to cameCase I will have to provide much more code to map them to different JSON names.